### DISCUSSION TITLE
Add world graph of all countries combined

### DISCUSSION THREAD

**Discussion by brendanwinter** [hodcroftlab/covariants#241] [Ideas]
To get a "big picture" of the global pandemic, would love a world graph that shows all countries combined.

**Comment by emmahodcroft** [comment]
@brendanwinter Thanks very much for your suggestion! This is something I've played around with in the past - the challenge is that the amount of sequencing per country varies quite a lot - so if I was just to lump all the numbers together, the graph would be totally dominated by the USA & UK, and not necessarily representative. One can try to control for this, but can't control for all the countries that are less able (or unable) to sequence - so I worry it gives a false sense of "the world". I'd be happy to hear suggestions for ways to try and overcome this, or make it very clear the limitations in a visualization, though!

**Reply by brendanwinter** [reply]
Ahh I see! For each country, could you multiply sequence frequency by reported cases that day to get estimates of daily cases by variant? You would still end up with a world graph dominated in proportion to cases, but that at least would be more accurate than being dominated by the amount of sequencing. Or, to avoid the US/UK dominance, perhaps regional graphs would work? In any case, thanks so much for your work, it's a lovely thing you've created!

**Reply by emmahodcroft** [reply]
Thank you! Yes, I'll consider if there's a way to do it, but I tend to try to keep CoVariants as data-focused as I can, so that there's as few caveats as possible. But feel free to make an issue, and perhaps we can think up a way to integrate it!
